#!/usr/bin/env python

PACKAGE = "project_1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

wh_enum = gen.enum([ gen.const("from_enc_velocity",      str_t, "from_enc_velocity", "input from wheels_states velocity"),
                                 gen.const("from_enc_position",     str_t, "from_enc_position", "input from wheels_states position")],
                               "The two method that can be chosen")
gen.add("method_input_wh_states", str_t, 0, "Method chosen for the input selection", "from_enc_position", edit_method=wh_enum)

exit(gen.generate(PACKAGE, "compute_VW", "wh_parameters"))